NEWS

```diff
@@ -2,6 +2,10 @@ systemd System and Service Manager
 
 CHANGES WITH 243 in spe:
 
+        * Enable ping(8) inside rootless Podman containers by turning on the
+          net.ipv4.ping_group_range parameter of the Linux kernel for all
+          groups.
```

Comment:
what key her is, is to say that this enables unprivileged programs of any kind to send ICMP ping requests, and if that's not desirable people should undo the sysctl change.

Reply:
Ok, reworded.